thin-replica-server/include/thin-replica-server/thin_replica_impl.hpp

```diff
@@ -59,17 +59,21 @@ struct ThinReplicaServerConfig {
   // the set of client IDs known to the TRS, used to authorize prospective
   // clients.
   std::unordered_set<std::string> client_id_set;
+  // the threshold after which metric aggregator is updated
+  uint16_t update_metrics_aggregator_thresh;
 
   ThinReplicaServerConfig(const bool is_insecure_trs_,
                           const std::string& tls_trs_cert_path_,
                           const concord::kvbc::IReader* rostorage_,
                           SubBufferList& subscriber_list_,
-                          std::unordered_set<std::string>& client_id_set_)
+                          std::unordered_set<std::string>& client_id_set_,
+                          uint16_t update_metrics_aggregator_thresh_)
```

Comment:
Let's use a default value `100` here. This parameter is not too important to the caller. Only if the default doesn't work then we are interested in modifying it.